packages/profile-picture/package.json

```diff
@@ -21,8 +21,10 @@
   "dependencies": {
     "@leavittsoftware/titanium-button": "^1.0.1",
```

Comment:
Can we make all of these peerDependencies ?

Reply:
Logically speaking, any web component could be declared as a peer dependency so there would only ever be one. It's not really what peer deps are for, but...

Reply:
I would suggest not to do this.  This component relies on this dependency, and it could end up not being included in  a project, and would but undo burden on the consumers.  Where lit-element should be included in all of our projects. If there are projects that have relied on titanium-elements to provide this dependency, then this will help us find this potential bug in projects.

Reply:
There could be an argument to do this because of the registering of components to the DOM, and potential for conflicts. This should not happen as lerna should keep all version in sync, but could happen if a consuming app has an old version of a titanium component and a newer version of a different titanium element and they both consume different versions of the same component.  I still suggest to wait and see if this becomes a problem.

Reply:
> There could be an argument to do this because of the registering of components to the DOM, and potential for conflicts. This should not happen as lerna should keep all versions in sync, but could happen if a consuming app has an old version of a titanium component and a newer version of a different titanium element and they both consume different versions of the same component. I still suggest to wait and see if this becomes a problem.

Yeah this actually happens all the time already.  You basically have to upgrade all titanium-components at once or your gonna be in trouble.  

That being said, I do agree peer dep. is not the correct move either.